mistletoe/html_renderer.py

```diff
@@ -205,7 +205,7 @@ def render_document(self, token: block_token.Document) -> str:
 
     @staticmethod
     def escape_html(raw: str) -> str:
-        return html.escape(raw).replace('&#x27;', "'")
+        return html.escape(raw)
```

Comment:
Sure that can be done. Setting `quote=False` for `render_raw_text` does however result in several specification tests failing because the spec tests expect `"` to be escaped as `&quot;` in regular text. We could resolve that by using the official CommonMark `spec_tests.py` test framework (which I am currently also working on rewriting).

Anyway I think that this performance concern should be addressed in a followup PR.

Reply:
OK, let it be as is for now, thank you.

> ... which I am currently also working on rewriting

Looks like you are really into this project now, which is great, but I'm not sure if I will be always able to keep up with your pace. ;)

Reply:
Ah sorry for the confusion, I meant in the [commonmark-spec](https://github.com/commonmark/commonmark-spec) repository ... not here.

Reply:
Never mind, thanks for the clarification. :)